testsuite/azure-arc-conformance.properties

```diff
@@ -0,0 +1,45 @@
+# Azure arc conformance test properties file
```

Comment:
Please also add a troubleshooting guide for partners in case of any kind of failures.
-> How to fetch launcher container logs
-> How to monitor launcher container logs (how to determine execution is complete)
-> kubectl cp commands to copy sonobuoy results if storage upload fails

Reply:
updated